packages/sdk/src/assets/README.md

```diff
@@ -0,0 +1,97 @@
+#### TODO: Move this content to our SDK docs website soon
```

Comment:
This is a great start! I do like the ordering. We will translate this to the doc sites.